packages/update-versions-json.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
+
+# `create_hash` creates a hash for a given file.
+function create_hash {
+  nix hash file --sri --type sha256 "$1" || exit 1
+}
+
+# `update` appends a new version entry to a given section.
+function update {
+  out=$(jq --arg NAME "$1" --arg HASH "$2" --arg VERSION "$VERSION" '.[$NAME] |= . + [{"version": $VERSION,hash: $HASH}]' ./packages/versions.json || exit 1)
```

Comment:
```suggestion
  out=$(
      jq --arg NAME "$1" \
          --arg HASH "$2" \
          --arg VERSION "$VERSION" \
          '.[$NAME] |= . + [{"version": $VERSION,hash: $HASH}]' \
          ./packages/versions.json
)
```

Format something like that for better readability